common/src/reserved.rs

```diff
@@ -0,0 +1,41 @@
+use crate::*;
+use serde::{Deserialize, Serialize};
+
+/// The partial set of the blockchain state which is reserved and protected.
+///
+/// It is stored in the reserved directory of the repository.
+/// Any transaction which modifies this state MUST produce a valid next one.
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
+pub struct ReservedState {
+    /// The genesis info. This must never be changed.
+    pub genesis_info: GenesisInfo,
+    /// The members.
+    pub members: Vec<Member>,
+    /// The leader order of the consensus rounds.
+    pub consensus_leader_order: Vec<usize>,
+    /// The semantic version of Simperby protocol for this network.
+    pub version: String,
+}
+
+impl ReservedState {
+    pub fn create_validator_set(&self) -> Result<Vec<(PublicKey, VotingPower)>, String> {
+        let mut validator_set = Vec::new();
+        for leader in &self.consensus_leader_order {
+            let member = &self.members.get(*leader).ok_or(format!(
```

Comment:
Is there any reason the lines are not aligned?

Reply:
it's probably just what rustfmt did